dialog/dialog_common.js

```diff
@@ -32,9 +32,9 @@ window.addEventListener('DOMContentLoaded', function () {
    * browser.windows.create won't show contents unless resized.
    * See https://bugzilla.mozilla.org/show_bug.cgi?id=1402110
    */
-  browser.windows.getCurrent((win) => {
-    browser.windows.update(win.id, {'width': win.width + 1});
-  });
+  // browser.windows.getCurrent((win) => {
```

Comment:
As documented, this is here as a workaround for a FF bug. Please don't simply remove this